vebit.lektorproject

```diff
@@ -17,6 +17,3 @@ target = rsync://vebit@vebit.xyz/var/www/virtual/vebit/html
 name = Deutsch
 primary = yes
 locale = de_DE
-
-[packages]
-lektor-scss = 1.3.8
```

Comment:
I liked the feature of lektor that if I release a new version of lektor-scss and write in here, that exactly this one will be used. And you don't have to hope that ``make install`` will be executed again.

But of course it's also a cool feature to modify the website from nix os.

Reply:
If there only would be an easy way to disable it on nixOS...
Maybe I can set a variable to redirect it to the users home directory instead of the system one. Basically I want to run ``pip install --user lekotr-scss==1,3,8``.

Reply:
I found a shell hook in the nixos wiki where this change is no longer necessary.